### PR TITLE
WIP: Product page

### PR DESCRIPTION
**Pull request by way2nizam**
I have added a product page.

product-page

### CODE REVIEW

File: styles/products.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  margin: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 1.5rem;
+}
+
+.product-card {
+  background-color: #fff;
+  max-width: 300px;
```
Comment (amarphule) [inline styles/products.css:11]: use `rem`

File: styles/products.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  outline: none;
+  font-size: 1.4rem;
+  font-weight: 600;
+  cursor: pointer;
+  padding: 1rem 2rem;
+  white-space: nowrap;
+  text-align: center;
+  width: 100%;
+  background-color: var(--color-grey-dark-3);
+  color: #fff;
```
Comment (amarphule) [inline styles/products.css:60]: create variable and use variable name

**Review by amarphule — CHANGES_REQUESTED** [review]
Looks good, some suggestions you can modify
1. Create badge on card items
2. Use cursor Pointer for card item
3. Make header fixed at the top

File: components/products.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+            </div>
+        </div>
+
+    </div>
+
+
+
+</body>
+
+</html>
```
Comment (kedark152) [inline components/products.html]: add extra empty line at the end of the file to avoid this red symbol.

File: components/products.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                <div class="product-card__title">Men Premium Jacket</div>
+                <h3 class="product-card__price">$2000</h3>
+
+                <button class="btn">Add to Card</button>
+
+            </div>
+        </div>
+        <div class="product-card">
+            <div class="product-card__cover">
+                <img class="img" src="../images/product.png" alt="">
```
Comment (kedark152) [inline components/products.html]: let there be some descriptive text for image in alt attribute.

**Review by kedark152 — COMMENTED** [review]
UI Looks good and code is written well.
Suggestion: Please add filter section and variations of products for eg: t-shirts, full shirts etc.

File: styles/products.css (hunk 1 above)
Comment (way2nizam) [reply styles/products.css:11]: thanks, I will fix.

File: styles/products.css (hunk 2 above)
Comment (way2nizam) [reply styles/products.css:60]: okay

File: components/products.html (hunk 4 above)
Comment (way2nizam) [reply components/products.html]: thanks, fixed

**Pull request merged**